liri.js

```diff
@@ -0,0 +1,207 @@
+//Keys to access twitter and spotify apis from an external file
+var keys = require("./keys.js");
+
+//external package to handle twitter api calls
+var Twitter = require('twitter');
+
+////external package to handle spotify api calls
+var Spotify = require('node-spotify-api');
+
+//external package to handle omdb api calls
+var request = require("request");
+
+//external package to handle files
+var fs = require("fs");
+
+var inquirerUserPromppt = require("inquirer");
+
+main();
+
+function main() { 
+inquirerUserPromppt
+.prompt([
+		{
+		  type: "checkbox",
+		  message: "Hi, my name is Liri. What do you want me search for you?",
+		  name: "commands",
+		  choices: 
+		  [
+		  	"my-tweets",
+		  	"spotify-this-song",
+		  	"movie-this",
+		  	"do-what-it-says",
+		  	"exit"
+		  ]
+		}
+	])
+	.then(function(inquirerResponse) {
+		console.log("You've chosen: "+inquirerResponse.commands+". Please follow instructions!");
+		var commands = JSON.stringify(inquirerResponse.commands);
+		switch (commands) {
+			case '["my-tweets"]':
+			console.log("twitter");
+			var inquirerTwitter = require("inquirer");
+			inquirerTwitter
+			.prompt([
+					{
+				      type: "input",
+				      message: "Please type a twitter username",
+				      name: "username"
+				    }
+				])
+			.then(function(inquirerResponse) {
+				twitter(inquirerResponse.username);
+			})
+			break;
+
+			case '["spotify-this-song"]':
+			var inquirerSpotify = require("inquirer");
+			inquirerSpotify
+			.prompt([
+					{
+				      type: "input",
+				      message: "Please the name of a song",
+				      name: "song"
+				    }
+				])
+			.then(function(inquirerResponse) {
+				spotify(inquirerResponse.song);
+			})
+			break;
+
+			case '["movie-this"]':
+			var inquirerMovie = require("inquirer");
+			inquirerMovie
+			.prompt([
+					{
+				      type: "input",
+				      message: "Please type a movie name",
+				      name: "movie"
+				    }
+				])
+			.then(function(inquirerResponse) {
+				movie(inquirerResponse.movie);
+			})
+			break;
+
+			case '["do-what-it-says"]':
+			doIt();
+			break;
+
+			default:
+			console.log("Bye!")
+	}
+	});
```

Comment:
Your function names should be a little better, like instead of "movie" try "movieSearch", instead of "doIt" try "readFileCommands"